parse_and_import_medicover.py

```
@@ -365,26 +444,33 @@ def main(
                         parsed_variant_data[value] = formatted_output
 
                 match = re.search(
-                    r"(?P<gm_number>GM[0-9]{2}_[0-9]+)", report, re.IGNORECASE
+                    r"(?P<gm_number>GM[0-9]{2}_[0-9]+)|(?P<sp_number>SP[0-9]{5}R[0-9]{4})", report, re.IGNORECASE
                 )
-
+                match_sp = re.search(
```

Ruff (0.14.7)

Local variable `match_sp` is assigned to but never used

Remove assignment to unused variable `match_sp`

(F841)